fix(comparison-content): correct Sentry open-source claims to reflect FSL licensing (#18346)
PostHog/posthog.com: contents/blog/best-sentry-alternatives.mdx

<details>
  <summary>Which Sentry alternatives are open source?</summary>

> **PostHog** is fully open source under the MIT license with a public roadmap. **Sentry** is self-hostable and source-available: its main web app uses the Functional Source License (FSL), which is not OSI-approved and converts to Apache-2.0 only after a two-year grace period. Most other alternatives like **Rollbar**, **Raygun**, **LogRocket**, and **AppSignal** are proprietary and cloud-only.

Reviewer: ```suggestion
**PostHog** is fully open source under the MIT license with a public roadmap. **Sentry** itself is self-hostable and source-available; its main web app uses the Functional Source License (FSL) and converts to Apache-2.0 after two years. Most other alternatives like **Rollbar**, **Raygun**, **LogRocket**, and **AppSignal** are proprietary and cloud-only.
```
